accepted/interleaved-declarations.md

```diff
@@ -98,16 +100,24 @@ The *current keyframe block* is the CSS keyframe block that was created by the
 innermost [execution of a style rule]. **This may be overridden by the
 [execution of a declaration].**
 
-## Declarations
+## Procedures
 
-### Semantics
+### Splitting the Current Parent If Necessary
 
-Add the following to the semantics for [executing a declaration] `declaration`
-before "Append `css` to `parent`":
+This procedure accepts no arguments and returns a style rule, keyframe block,
+at-rule, or null.
 
-[executing a declaration]: ../spec/declarations.md#semantics
+* Let `parent` be the [current style rule], [keyframe block], or at-rule if one
+  exists; or the innermost if multiple exist.
+
+  [current style rule]: ../spec/style-rules.md#current-style-rule
+  [keyframe block]: ../spec/style-rules.md#current-style-rule
 
-* If `parent` isn't the last statement in its parent:
+* If `parent` is null, return null.
+
+* Otherwise, if `parent` is the last statement in its parent, return `parent`.
```

Comment:
Something like `If parent is the last statement within its container, return parent` would be more clear, or if we want to refer to the grandparent something like `If parent is the last child of its parent, return parent.`